react-frontend/src/coreComponents/ButtonCoreComponent.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Button } from "react-bootstrap";
+import PropTypes from "prop-types";
+
+const ButtonCoreComponent = (props) => {
+  const { value, className, type, onClick, variant, size } = props;
+  return (
+    <Button
+      className={className}
+      type={type}
+      value={value}
```

Comment:
is there any difference between button value and button name text? if there is should we use value prop at both places?